public/app/features/explore/TraceView/components/TracePageHeader/TracePageHeader.tsx

```diff
@@ -110,76 +110,184 @@ export const TracePageHeader = memo((props: TracePageHeaderProps) => {
     }
   }
 
-  const urlTooltip = (url: string) => {
-    return (
-      <>
-        <div>
-          <Trans
-            i18nKey="explore.trace-page-header.tooltip-url"
-            values={{
-              url: 'http.url',
-              target: 'http.target',
-              path: 'http.path',
-            }}
-          >
-            {'{{url}}'} or {'{{target}}'} or {'{{path}}'}
-          </Trans>
-        </div>
-        <div>({url})</div>
-      </>
-    );
+  const copyTraceId = () => {
+    navigator.clipboard.writeText(trace.traceID);
+    setCopyTraceIdClicked(true);
+    setTimeout(() => {
+      setCopyTraceIdClicked(false);
+    }, 5000);
+  };
+
+  const exportTrace = () => {
+    const traceFormat = downloadTraceAsJson(data, 'Trace-' + trace.traceID.substring(trace.traceID.length - 6));
+    reportInteraction('grafana_traces_download_traces_clicked', {
+      app,
+      grafana_version: config.buildInfo.version,
+      trace_format: traceFormat,
+      location: 'trace-view',
+    });
   };
 
+  const shareDropdownMenu = (
```

Comment:
Out of curiosity - why do we put all items into a single dropdown? In designs they are all separated. I wonder if it's gonna be clear to customers that exporting in under share button now 🤔